tests/test_level/test_level_08.py

```diff
@@ -826,21 +830,27 @@ def test_double_if_pressed(self):
             print 'second key'""")
 
         expected = HedyTester.dedent("""\
-          while not pygame_end:
-            pygame.display.update()
-            event = pygame.event.wait()
-            if event.type == pygame.QUIT:
-              pygame_end = True
-              pygame.quit()
+        pygame_end = False
+        while not pygame_end:
+          pygame.display.update()
+          event = pygame.event.wait()
+          if event.type == pygame.QUIT:
+            pygame_end = True
+            pygame.quit()
+            break
+          if event.type == pygame.KEYDOWN:
+            if event.unicode != 'x':
+                pygame_end = True
+            if event.unicode == 'x':
+              print(f'first key')
               break
-            if event.type == pygame.KEYDOWN:
-              if event.unicode == 'x':
-                print(f'first key')
-                break
-            if event.type == pygame.KEYDOWN:
-              if event.unicode == 'y':
-                print(f'second key')
```

Comment:
I see that you changed the tests but didn't add one with command in between them?

Reply:
I completely forgot to add that test (I guess I was tired after changing all tests😅), thanks for adding the test @jpelay !!